sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1492,6 +1492,15 @@ object SQLConf {
         "This usually speeds up commands that need to list many directories.")
       .booleanConf
       .createWithDefault(true)
+
+  val BYPASS_PARSER_FOR_EMPTY_SCHEMA =
+    buildConf("spark.sql.legacy.bypassParserForEmptySchema")
```

Comment:
If no behavior change, do we still need this conf?

Reply:
It seems we don't need it anymore